src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlTypes/SqlFileStream.Windows.cs

```diff
@@ -689,8 +689,13 @@ static private string InitializeNtPath(string path)
             // Ensure we have validated and normalized the path before
             AssertPathFormat(path);
             string uniqueId = Guid.NewGuid().ToString("N");
-            return System.IO.PathInternal.IsDeviceUNC(path) ? string.Format(CultureInfo.InvariantCulture, @"{0}\{1}", path.Replace(@"\\.", @"\??"), uniqueId)
-                                                            : string.Format(CultureInfo.InvariantCulture, @"\??\UNC\{0}\{1}", path.Trim('\\'), uniqueId);
+#if NETSTANDARD
+            return System.IO.PathInternal.IsDeviceUNC(path.AsSpan())
+#else
+            return System.IO.PathInternal.IsDeviceUNC(path)
+#endif
+                ? string.Format(CultureInfo.InvariantCulture, @"{0}\{1}", path.Replace(@"\\.", @"\??"), uniqueId)
+                : string.Format(CultureInfo.InvariantCulture, @"\??\UNC\{0}\{1}", path.Trim('\\'), uniqueId);
```

Comment:
I have seen usage of `string.Format` and string `interpolation` in this changes. Can we stay with string interpolation ?
string.Format has some limitation which has been addressed in string interpolation.
 - Every time String.Format is called, it needs to parse the composite format string to find all the literal portions of the text, all of the format items, and their specifiers and alignments; somewhat ironically in the case of string interpolation, the C# compiler already had to do such parsing in order to parse the interpolated string and generate the String.Format, yet it has to be done again at run-time for each call.
- These APIs all accept arguments typed as System.Object, which means that any value types end up getting boxed in order to be passed in as an argument.
- There are String.Format overloads that accept up to three individual arguments, but for cases where more than three are needed, there’s a catch-all overload that accepts a params Object[]. That means any number of arguments more than three allocates an array.

More could be read [here](https://devblogs.microsoft.com/dotnet/string-interpolation-in-c-10-and-net-6/)

Reply:
I didn't modify the file, it's as-is.